### ISSUE TITLE
List is good but it is practically useless

### ISSUE THREAD

**Issue by snapfast** [columndeeply/hosts#10]
It blocks too many urls and many important urls also.

Can you please create another list which has google.com and other websites that are allowed ?

**Comment by columndeeply** [comment]
What sites did you find that shouldn't be blocked? I'm using the list right now and Google is working fine, it's not blocked.

**Comment by Macaquinyo** [comment]
I agree. I'm using the lists with pihole, and it blocks some sites it shouldn't

For examample, it blocks "m.facebook.com". I think this is an error, for the desktop site is not blocked but the smartphone site is. "www.fbcdn.net" is also on the list. This is the domain where facebook keeps its images.

"www.twitter.com" and "x.com" also. This might be intentional, since porn is served on twitter. But "pbs.twimg.com", where twitter stores its images, is also blocked. Many sites embeed twitter images, and this breaks that.
Some nitter (alternative frontend for twitter) instances are blocked, v.gr. "nitter.poast.org"

"reddit.com" is blocked, like twitter might be intentional, but "old.reddit.com" is not, so you can still access.

"alicdn.com" is blocked. This is the alibaba cloud service. Aliexpress serves its images from "ae01.alicdn.com"

"i0.wp.com". Where wordpress images live. No images will load in wordpress sites.

"www.mediafire.com", a file hosting site, is blocked. Other file hosting sites, like mega, are not blocked.

"imgur.com" many, many sites and user host their images on imgur. Again, might be intentional.

**Comment by Macaquinyo** [comment]
Found a handy list with the top 10 million domains (https://www.domcop.com/top-10-million-websites). Pornhub is the top pornography site per many other lists, so I'll take every site that's higher in the list than pornhub and see it it apears on the list.

**Comment by Macaquinyo** [comment]
OK, so, I grabbed the top 10320 most visited sites, per https://www.domcop.com/top-10-million-websites, removed every site that was present in other porn filtering lists ([this one](https://github.com/4skinSkywalker/anti-porn-hosts-file) and the one [here](https://github.com/StevenBlack/hosts?tab=readme-ov-file)), checked which sites where both in the top and in your list (both adding www. and not adding it), and here they are. In total, 739.

There's still two things to do. 
First, doublecheck that no porn site slipped trought the cracks. I've been looking and I've not found one.
Second, determine which ones should be removed from the list, and which ones should stay or be added into a second list of common sites that also serve porn, so users can know which mainstream sites are blocked.

**Comment by columndeeply** [comment]
Thanks for lists @Macaquinyo. I'll whitelist alibaba, mediafire and wordpress' CDN (and facebook if it doesn't allow porn, I'll check later), but I'd like to keep twitter/reddit/imgur blocked since all of them allow porn and last time I checked it wasn't hard to find explicit stuff just by browsing those sites.

I'm not against moving them to a separate list but only if there's more stuff that would fit. I don't want to create a separate list for three or four domains. At that point I'd rather users whitelist them manually if they need to access them.

Not sure why old.reddit.com isn't blocked but it should be... The nitter domains being blocked is intentional.

If you have more domains you think should be moved to a "popular sites that allow porn" list please let me know and we can see if it's worth it to create a separate list.

**Comment by robertgro** [comment]
> It blocks too many urls and many important urls also.
> 
> Can you please create another list which has google.com and other websites that are allowed ?

Entirely not true.

None of the provided blacklists has an aforementioned whitelist url line in it. Search for yourself, the files are publicly available.

Who else landed here, dealing with a large windows 10 hosts file and falsly claiming here the blacklists would contain a whitelist url like "m.facebook.com"?
